### PR TITLE
Air leakage type "unit total"

### PR DESCRIPTION
**Pull request by joseph-robertson**
## Pull Request Description

This PR fixed a bug that resulted in the modeled ACH50 infiltration values of the units being larger than the sampled ACH50 for the building. The update is to change options_lookup and ResStockArguments to use **unit total** air leakage type instead of the current **unit exterior only** type w/ infiltration adjustment approach. This change provides the infiltration value sampled as the total infiltration from the exterior and other neighboring units. The infiltration to the exterior is then calculated as a percentage of the exterior surfaces. This change will reduce the modeled infiltration for MF units. The modeled infiltration to the exterior (`build_existing_model.air_leakage_to_outside_ach_50`) is now reported in the results.

### Updates
Remove the "Infiltration adjustment for SFA/MF units" block of code from ResStockArguments.
Update BuildExistingModel to register adjusted total infiltration ACH50 value.
Some related plots in: https://github.com/NREL/resstock/issues/728

Some clarification (need documentation update): 
- [x] Sampled ACH 50 is the total unit leakage from outside and adjacent units
  - Add/update "Assumptions" in the Infiltration.tsv file?
- [x] The output leakage is the simulated leakage to the outside
  - We've updated `data/dictionary/inputs.csv` with an `build_existing_model.air_leakage_to_outside_ach_50` entry along with a description: "Total infiltration to the dwelling unit adjusted by ratio of exterior envelope surface area to total envelope surface area."

image
image
image

## Checklist

Not all may apply:

- [ ] ~Tests (and test files) have been updated~
- [x] Documentation has been updated
  - [ ] ~If related to resstock-estimation, checklist includes [data dictionary](https://github.com/NREL/resstock/tree/develop/resources/data/dictionary), [source report](https://github.com/NREL/resstock/tree/develop/project_national/resources/source_report.csv), [options saturation](https://github.com/NREL/resstock/tree/develop/project_national/resources/options_saturations.csv), [options_lookup](https://github.com/NREL/resstock/blob/develop/resources/options_lookup.tsv).~
- [x] Add to the [changelog_dev.rst file](https://github.com/NREL/resstock/tree/develop/docs/read_the_docs/source/changelog/changelog_dev.rst)
- [x] `openstudio tasks.rb update_measures` has been run
- [x] No unexpected regression test changes on CI (checked comparison artifacts)

### CODE REVIEW

File: measures/ApplyUpgrade/resources/constants.rb [hunk 1]
```diff
@@ -28,6 +28,7 @@ def self.CostMultiplierChoices
       'Duct Unconditioned Surface Area (ft^2)',
       'Rim Joist Area, Above-Grade, Exterior (ft^2)',
       'Slab Perimeter, Exposed, Conditioned (ft)',
+      'Air Infiltration (ACH50)',
```
Comment (shorowit) [inline measures/ApplyUpgrade/resources/constants.rb:31]: There is already cost multiplier (`'Floor Area, Conditioned * Infiltration Reduction (ft^2 * Delta ACH50)'`) intended to be used for air sealing measures. How would you use the ACH50 directly for costing?

Comment (joseph-robertson) [reply measures/ApplyUpgrade/resources/constants.rb:31]: This isn't permanent. It's the avenue I'm taking for reporting the ACH50 value (i.e., the HPXML file needs to be built and defaulted before I can get/calculate the ACH50 value).

**Comment by joseph-robertson** [comment]
Next steps here are, in order:
* look at energy impacts (especially heating, cooling)
* compare our distributions to ResDB
* replace the new cost multiplier approach with, e.g,, `build_existing_model.air_infiltration_leakage_to_outside_xxx`

**Comment by joseph-robertson** [comment]
Questions from meeting:
* How do we compare new ResStock distributions to what we have available on ResDB? Qualitative comparison?
* Convert from Normalized leakage (NL) to ACH50?

**Comment by afontani** [comment]
> Questions from meeting:
> 
> * How do we compare new ResStock distributions to what we have available on ResDB? Qualitative comparison?
> * Convert from Normalized leakage (NL) to ACH50?

Maybe compare the relative multipliers for the different building types (see the last two columns)
Screenshot 2024-07-09 at 10 29 50 AM

We are generally the same as ResDB for the whole stock.
Screenshot 2024-07-09 at 10 30 33 AM

**Comment by joseph-robertson** [comment]
Meeting notes:
- [ ] Use the data we generated the ACH50 plots from to get percentiles. Compare these to the ResDB SFD numbers times multipliers in the table above.
- [ ] This PR is a bugfix PR. There may be a follow-up PR to adjust distributions to better match ResDB.

**Comment by joseph-robertson** [comment]
Infiltration sources | 5th | 50th | 95th | Segment
-- | -- | -- | -- | --
ResDB | 4 | 12 | 35 | Mostly single-family detached
ResDB | 5.2 | 15.6 | 45.5 | Single-Family Attached Homes
ResDB | 2.8 | 8.4 | 24.5 | Multi-Family Homes
ResDB | 6.16 | 18.48 | 53.9 | Manufactured Homes


Infiltration sources | 5th | 50th | 95th | Segment
-- | -- | -- | -- | --
ResStock | 3 | 15 | 40 | U.S. residential stock
ResStock | 4 | 15 | 40 | Single-Family Detached
ResStock | 3.436187 | 10.19865 | 26.85008 | Single-Family Attached
ResStock | 1.0734 | 8.1402 | 26.8074 | Multi-Family

**Comment by ekpresent** [comment]
Sorry, this PR caught my attention and I have a couple naive questions.
1. Are there revised plots available like the ones @joseph-robertson posted on June 13?
2. Was an energy results comparison ever completed? (sorry if I just don't know where to look for it) - I'd be interested in segmentation by at least building type and climate zone.

**Comment by joseph-robertson** [comment]
Plan is to:
* run 10K downselected to only SFA/MF on develop
* run same datapoints using this feature branch
* plots of heating/cooling, sliced by building type and climate zone

**Comment by rajeee** [comment]
These enduses are going up for some reason. Might be worth finding out why.

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:////Users/radhikar/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip.htm">
<link rel=File-List
href="file:////Users/radhikar/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip_filelist.xml">
<style>

</style>
</head>

<body link="#467886" vlink="#96607D">


  | geometry_building_type_recs | base | feature | % Error
-- | -- | -- | -- | --
end_use_electricity_cooling_fans_pumps_m_btu | Single-Family Attached | 0.75801682 | 0.78837944 | 4.00553338
end_use_electricity_cooling_fans_pumps_m_btu | Multi-Family with 5+ Units | 0.46667142 | 0.47255323 | 1.26037515
end_use_electricity_cooling_fans_pumps_m_btu | Multi-Family with 2 - 4 Units | 0.23117381 | 0.23384424 | 1.15516063
end_use_electricity_cooling_fans_pumps_m_btu | All | 1.4115813 | 1.4140257 | 0.1731675



</body>

</html>

**Comment by joseph-robertson** [comment]
> These enduses are going up for some reason. Might be worth finding out why.
> 
>  	geometry_building_type_recs	base	feature	% Error
> end_use_electricity_cooling_fans_pumps_m_btu	Single-Family Attached	0.75801682	0.78837944	4.00553338
> end_use_electricity_cooling_fans_pumps_m_btu	Multi-Family with 5+ Units	0.46667142	0.47255323	1.26037515
> end_use_electricity_cooling_fans_pumps_m_btu	Multi-Family with 2 - 4 Units	0.23117381	0.23384424	1.15516063
> end_use_electricity_cooling_fans_pumps_m_btu	All	1.4115813	1.4140257	0.1731675

For example, ID=21. Modeled ACH50 for this MF unit changes from 8.09 to 0.59. HeatingCapacity (furnace) changes from 6188 to 1860. CoolingCapacity (mini-split air conditioner) changes from 8097 to 6066. Indeed cooling increases slightly. I'm looking into this.

**Comment by joseph-robertson** [comment]
> > These enduses are going up for some reason. Might be worth finding out why.
> >  	geometry_building_type_recs	base	feature	% Error
> > end_use_electricity_cooling_fans_pumps_m_btu	Single-Family Attached	0.75801682	0.78837944	4.00553338
> > end_use_electricity_cooling_fans_pumps_m_btu	Multi-Family with 5+ Units	0.46667142	0.47255323	1.26037515
> > end_use_electricity_cooling_fans_pumps_m_btu	Multi-Family with 2 - 4 Units	0.23117381	0.23384424	1.15516063
> > end_use_electricity_cooling_fans_pumps_m_btu	All	1.4115813	1.4140257	0.1731675
> 
> For example, ID=21. Modeled ACH50 for this MF unit changes from 8.09 to 0.59. HeatingCapacity (furnace) changes from 6188 to 1860. CoolingCapacity (mini-split air conditioner) changes from 8097 to 6066. Indeed cooling increases slightly. We'll need to look into this. @shorowit do you have any insight here?

This could be due to a high amount of internal gains -- infiltration was previously aiding the cooling. Should check number of occupants, plug load multiplier.

**Comment by joseph-robertson** [comment]
> > > These enduses are going up for some reason. Might be worth finding out why.
> > >  	geometry_building_type_recs	base	feature	% Error
> > > end_use_electricity_cooling_fans_pumps_m_btu	Single-Family Attached	0.75801682	0.78837944	4.00553338
> > > end_use_electricity_cooling_fans_pumps_m_btu	Multi-Family with 5+ Units	0.46667142	0.47255323	1.26037515
> > > end_use_electricity_cooling_fans_pumps_m_btu	Multi-Family with 2 - 4 Units	0.23117381	0.23384424	1.15516063
> > > end_use_electricity_cooling_fans_pumps_m_btu	All	1.4115813	1.4140257	0.1731675
> > 
> > 
> > For example, ID=21. Modeled ACH50 for this MF unit changes from 8.09 to 0.59. HeatingCapacity (furnace) changes from 6188 to 1860. CoolingCapacity (mini-split air conditioner) changes from 8097 to 6066. Indeed cooling increases slightly. We'll need to look into this. @shorowit do you have any insight here?
> 
> This could be due to a high amount of internal gains -- infiltration was previously aiding the cooling. Should check number of occupants, plug load multiplier.

ID=21 is:
* unit in a 183-unit apartment building
* 2 bed, 1138 cfa
* MD, North Bethesda
* 2 occupants
* electric furnace, shared fan coil
* plug load multiplier 1.06

**Review by afontani — APPROVED** [review]

**Pull request merged**